### PR TITLE
GEOMESA-2880 Check and install missing dependencies in CLI tools

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
need to update install docs

File: geomesa-accumulo/geomesa-accumulo-tools/pom.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@
                         <argument>org.locationtech.geomesa.accumulo.tools.AccumuloRunner</argument>
                         <argument>help</argument>
                         <argument>--autocomplete-function</argument>
-                        <argument>${project.build.directory}/autocomplete.sh,geomesa</argument>
+                        <argument>${project.build.directory}/autocomplete.sh,geomesa-accumulo</argument>
```
Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-tools/pom.xml:162]: Where did you land on the autocomplete.sh?

File: geomesa-cassandra/geomesa-cassandra-tools/bin/geomesa-cassandra [hunk 2]
```diff
@@ -7,55 +7,74 @@
...
-#
-# configure runs the geomesa configuration, sets both the %%gmtools.dist.name%%_HOME and PATH vars
-# classpath prints the class path, ie list of jars we are using
-# else is running actual commands in the tools
+# set java process options - do this after set classpath b/c that can modify JAVA_LIBRARY_PATH
+if [[ $1 = debug ]]; then
+  GEOMESA_OPTs="$(get_options "debug")"
+  shift 1
+else
+  GEOMESA_OPTs="$(get_options)"
```
Comment (jnh5y) [inline geomesa-cassandra/geomesa-cassandra-tools/bin/geomesa-cassandra:51]: Should the final 's' be capital?

File: geomesa-accumulo/geomesa-accumulo-tools/pom.xml (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-accumulo/geomesa-accumulo-tools/pom.xml:162]: still there. this fixes it for accumulo

File: geomesa-cassandra/geomesa-cassandra-tools/bin/geomesa-cassandra (hunk 2 above)
Comment (elahrvivaz) [reply geomesa-cassandra/geomesa-cassandra-tools/bin/geomesa-cassandra:51]: ooh, yes good catch

**Comment by jnh5y** [comment]
Do reference to install-jline and install-jai.sh need to be updated?

**Comment by elahrvivaz** [comment]
i think i updated them all... git grep is clean

**Comment by elahrvivaz** [comment]
I verified that all the dependencies already have approved cqs from when we bundled hbase-server 1.4 in previous versions.

**Comment by elahrvivaz** [comment]
ready for final review and merge

**Comment by jnh5y** [comment]
Merged as 43dafe1125cdc5dc3e7ed1168849be5f0c6c37ab.

**Pull request closed without merging**